data_validation/clients.py

```diff
@@ -181,7 +182,7 @@ def get_all_tables(client, allowed_schemas=None):
         try:
             tables = list_tables(client, schema_name)
         except Exception as e:
-            print(f"List Tables Error: {schema_name} -> {e}")
+            logging.exception(f"List Tables Error: {schema_name} -> {e}")
             continue
```

Comment:
My understanding is this shouldn't be an exception level. We could use warning or info

Reply:
Warning would make sense to me